x-pack/test/functional/apps/endpoint/alerts.ts

```diff
@@ -28,12 +28,31 @@ export default function({ getPageObjects, getService }: FtrProviderContext) {
     it('includes Alert list data grid', async () => {
       await testSubjects.existOrFail('alertListGrid');
     });
-    it('updates the url upon submitting a new search bar query', async () => {
-      await pageObjects.endpointAlerts.enterSearchBarQuery();
-      await pageObjects.endpointAlerts.submitSearchBarFilter();
-      const currentUrl = await browser.getCurrentUrl();
-      expect(currentUrl).to.contain('query=');
-      expect(currentUrl).to.contain('date_range=');
+    describe('when submitting a new bar query', () => {
+      before(async () => {
+        await pageObjects.endpointAlerts.enterSearchBarQuery('test query');
+        await pageObjects.endpointAlerts.submitSearchBarFilter();
+      });
+      it('should update the url correctly', async () => {
+        const currentUrl = await browser.getCurrentUrl();
+        expect(currentUrl).to.contain('query=');
+        expect(currentUrl).to.contain('date_range=');
+      });
+      after(async () => {
+        await pageObjects.endpointAlerts.enterSearchBarQuery('');
```

Comment:
Why do you need this? is the page not being reloaded after each test?

Reply:
it's not being reloaded after each test, seemingly only for each file, so this just clears out the state so as to not impede further test with narrowed filter conditions